tests/e2e/utils/flows.js

```diff
@@ -43,6 +43,9 @@ const MY_ACCOUNT_ADDRESSES = baseUrl + 'my-account/edit-address';
 const MY_ACCOUNT_ACCOUNT_DETAILS = baseUrl + 'my-account/edit-account';
 const MY_ACCOUNT_PAYMENT_METHODS = baseUrl + 'my-account/payment-methods';
 
+const WP_ADMIN_SINGLE_CPT_VIEW = ( postId ) =>
```

Comment:
Makes sense! For this, I selected the name following the convention that is used in https://www.npmjs.com/package/@woocommerce/e2e-utils. I am open to changing it, I think my only thought on `WP_ADMIN_SINGLE_POST_VIEW` might be the assumption that it for linking to regular blog posts, but it could definitely be I am reading into it a bit too much and it would be fine.

Reply:
Sounds good! Thanks for the feedback!